app/src/main/res/layout/fragment_ai_proposition.xml

```diff
@@ -59,7 +59,7 @@
                 android:visibility="gone"
                 app:icon="@drawable/ic_warning"
                 app:showCloseIcon="true"
-                tools:text="@string/aiErrorTooManyRequests"
+                tools:title="@string/aiErrorTooManyRequests"
```

Comment:
The already existing InformationBlockView should use the description to stay coherent with the previous style